src/main/java/teammates/common/datatransfer/attributes/FeedbackSessionAttributes.java

```diff
@@ -227,7 +227,14 @@ public boolean isValid() {
         return getInvalidityInfo().isEmpty();
     }
 
+    /**
+     * Returns {@code true} if the session is closed after the given hours,
+     * {@code false} otherwise, or if the startTime or endTime is {@code null}.
+     */
     public boolean isClosedAfter(int hours) {
+        if (endTime == null || startTime == null) {
+            return false;
+        }
```

Comment:
I'm thinking we can use `Assumption.assertNotNull` instead to ensure that these methods are not called for private sessions. If we go this way, we don't need to talk about the null case in the javadoc.

Reply:
Okay.

Reply:
Should I use `Assumption.assertNotNull` for all methods? Like `isClosed()`, `isInGracePeriod`, `isOpened()` as these methods are not meant for private sessions as well?

Reply:
Ideally yes, but need to check all their usages first.

Reply:
Sorry to interrupt, but can we use `Preconditions.checkNotNull` if we are doing this? I think we should prefer to stop using the `Assumption` class.

Reply:
@whipermr5 @LiHaoTan  If we are going this way, then I have no problem in doing so. But just out of curiosity, what is the difference between the two?

Reply:
@sukanta-27 just for your information.

The assumption class is copied from Junit so the semantics is not the same as what we want.

Java already has `Objects#requireNonNull` which throws `NullPointerException`. Whether we agree with that is the exception to be thrown that is the Java conventions and it has become so common that we need good reason to throw another exception 

The method also intentionally allow chaining.

Using `Preconditions` is just for consistency.